api/v1alpha2/mysqlserver_types.go

```diff
@@ -33,7 +33,7 @@ type MySQLServerSpec struct {
 	// If AdminSecret is specified but a secret with the given name is not found in the same namespace
 	// as the MySQLServer, then reconciliation will block until the secret is created.
 	// If this is not specified, a username and password will be automatically generated.
-	AdminSecret            string               `json:"adminSecret,omitempty"`
+	AdminSecret string `json:"adminSecret,omitempty"`
```

Comment:
oops

Reply:
The fact that the old ASO build lets me merge this angers me... maybe will make a fix to that.